--- setup.py.bak	2012-02-02 17:07:01.072272403 -0500
+++ setup.py	2012-02-02 17:07:24.255109806 -0500
@@ -38,6 +38,4 @@
 
     packages=['psshlib'],
     scripts = [os.path.join("bin", p) for p in ["pssh", "pnuke", "prsync", "pslurp", "pscp", "pssh-askpass"]],
-    data_files=[('man/man1', ['man/man1/pssh.1', 'man/man1/pscp.1',
-        'man/man1/prsync.1', 'man/man1/pslurp.1', 'man/man1/pnuke.1'])],
     )
